packages/ui/tailwind.config.js

```diff
@@ -65,6 +132,18 @@ export default {
         'accordion-down': 'accordion-down 0.2s ease-out',
         'accordion-up': 'accordion-up 0.2s ease-out',
       },
+      fontFamily: {
+        sans: platformSelect({
+          android: ['Anakotmai_500Medium', 'sans-serif'],
+          ios: ['Anakotmai-Medium', "'system font'"],
+          default: ['Anakotmai-Medium', 'ui-sans-serif'],
+        }),
+        serif: platformSelect({
+          android: ['NotoSansThaiLooped_500Medium', 'serif'],
+          ios: ['NotoSansThaiLooped-Medium', 'Georgia'],
+          default: ['NotoSansThaiLooped-Medium', 'serif'],
```

Comment:
It seems that font did not load in IOS platform, do we need to specify font family `Anakotmai-500Medium` and `NotoSansThaiLooped-500Medium`

```suggestion
          android: ['Anakotmai_500Medium', 'sans-serif'],
          ios: ['Anakotmai-500Medium', "'system font'"],
          default: ['Anakotmai-500Medium', 'ui-sans-serif'],
        }),
        serif: platformSelect({
          android: ['NotoSansThaiLooped_500Medium', 'serif'],
          ios: ['NotoSansThaiLooped-500Medium', 'Georgia'],
          default: ['NotoSansThaiLooped-500Medium', 'serif'],
```

Reply:
Hmm Maybe it didn't load on first render
I'll test again
FYI Font name varies between platform 
https://docs.expo.dev/develop/user-interface/fonts/#use-google-fonts

Reply:
@miello Can you check again
To reproduce, try clear the cache with `npx expo start --clear`